### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by optimumtact**
Better descriptions of each role

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -22,20 +22,13 @@ There is an open list of approachable issues for [your inspiration here](https:/
...
-
-Project Managers are responsible for recruiting and firing maintainers, enforcing coding standards, and reverting changes that should have not been committed. Project Managers are assigned by Project Leads. On things that Project Managers disagree on they are to refer to the Project Leads for advice. It is encouraged that if you do not want to waste time working on a feature, that might be denied, that you ask a Project Manager first.
+Don't fuck with them they're straight up killers.
 
 **Maintainers**
 
-Maintainers are quality control. If a proposed pull request does not meet the mentioned quality specifications then it can be closed if you fail to satisfy them. Maintainers are required to give a reason for closing the pull request.
-
-Maintainers can revert your changes if they feel they are not worth maintaining or if they did not live up to the quality specifications.
+Maintainers will argue about your pr in their secret irc channel and then it will be merged and then reverted in some kind of metawar you can barely comprehend, you can attempt to improve this process by using the 8b command in their channel to convince them
```
Comment (Jordie0608) [inline .github/CONTRIBUTING.md]: What secret irc channel?

Comment (optimumtact) [inline .github/CONTRIBUTING.md]: Fry is on to you

**Review by Cyberboss — APPROVED** [review]

**Comment by Cyberboss** [comment]
I'm too memed out today...

**Comment by kevinz000** [comment]
oranges is right!!!

**Pull request closed without merging**